packages/api/src/router/appSessions.ts

```diff
@@ -195,7 +154,7 @@ export const appSessionsRouter = router({
 
       const appSession = {
         data: {
-          id: nanoid(16),
+          id: chatId,
```

Comment:
Looks like for a while there were separate ids for app sesssion and chat ... though they definitely used to be the same. 

It feels as though we might pull all the aila related sessions out of app sessions table at some point, but for me it feels easier for them to be the same for now.